internal/serverinstall/k8s.go

```diff
@@ -461,16 +465,20 @@ func (i *K8sInstaller) Upgrade(
 		i.Config.Namespace = "default"
 	}
 
+	if i.Config.Timeout == 0 {
+		i.Config.Timeout = 300
+	}
```

Comment:
This shouldn't be required, we can set a default over in the flag where it's defined!